lib/svg_sprite/svg_sprite.rb

```diff
@@ -188,39 +188,37 @@ module SvgSprite
   SVG_SPRITE_PATHS = Dir.glob(["#{Rails.root}/vendor/assets/svg-icons/**/*.svg",
                                "#{Rails.root}/plugins/*/svg-icons/*.svg"])
 
-  def self.svg_sprite_cache
-    @svg_sprite_cache ||= DistributedCache.new('svg_sprite')
+  def self.all_icons(theme_ids)
+    get_set_cache("icons_#{Theme.transform_ids(theme_ids).join(',')}") do
+      Set.new()
+        .merge(settings_icons)
+        .merge(plugin_icons)
+        .merge(badge_icons)
+        .merge(group_icons)
+        .merge(theme_icons(theme_ids))
+        .delete_if { |i| i.blank? || i.include?("/") }
+        .map! { |i| process(i.dup) }
+        .merge(SVG_ICONS)
+        .sort
+    end
   end
 
-  def self.all_icons
-    Set.new()
-      .merge(settings_icons)
-      .merge(plugin_icons)
-      .merge(badge_icons)
-      .merge(group_icons)
-      .merge(theme_icons)
-      .delete_if { |i| i.blank? || i.include?("/") }
-      .map! { |i| process(i.dup) }
-      .merge(SVG_ICONS)
-      .sort
+  def self.version(theme_ids)
+    get_set_cache "version_#{Theme.transform_ids(theme_ids).join(',')}" do
```

Comment:
We should be consistent with using `(` and `)`

```suggestion
    get_set_cache("version_#{Theme.transform_ids(theme_ids).join(',')}") do
```